versioned_docs/version-v0.11.0/int/quickstart/quickstart-alone.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_position: 4
+description: Run all nodes in a distributed validator cluster
+---
+
+# Run a cluster alone
+
+:::caution
+Charon is in an early alpha state and is not ready to be run on mainnet
+:::
+
+1. Clone the [charon-distributed-validator-cluster](https://github.com/ObolNetwork/charon-distributed-validator-cluster) template repo and `cd` into the directory.
+
+   ```sh
+   # Clone the repo
+   git clone https://github.com/ObolNetwork/charon-distributed-validator-cluster.git
+
+   # Change directory
+   cd charon-distributed-validator-cluster/
+   ```
+
+1. Prepare the environment variables
+
+   ```sh
+   # Copy the sample environment variables
+   cp .env.sample .env
+   ```
+
+   For simplicities sake, this repo is configured to work with a remote Beacon node such as one from [Infura](https://infura.io/).
+
+   Create an Eth2 project and copy the `https` URL, make sure Prater is selected in dropdown of ENDPOINTS:
+
+   ![Example Infura API Endpoint](/img/example-infura-details.png)
+
+   Replace the placeholder value of `CHARON_BEACON_NODE_ENDPOINT` in your newly created `.env` file with this URL.
```

Comment:
Remove references to infura. Can sync with the cdvc repo readme (https://github.com/ObolNetwork/charon-distributed-validator-cluster#quickstart)